### ISSUE TITLE
class Edge_Module, In forward function,  edge not be used??

### ISSUE THREAD

**Issue by whiteinblue** [ziwei-zh/CorrPM#6]
def forward(self, x1, x2, x3):
        _, _, h, w = x1.size()

        edge1_fea = self.conv1(x1)
        edge1 = self.conv4(edge1_fea)
        edge2_fea = self.conv2(x2)
        edge2 = self.conv4(edge2_fea)
        edge3_fea = self.conv3(x3)
        edge3 = self.conv4(edge3_fea)

        edge2_fea = F.interpolate(edge2_fea, size=(h, w), mode='bilinear', align_corners=True)
        edge3_fea = F.interpolate(edge3_fea, size=(h, w), mode='bilinear', align_corners=True)
        edge2 = F.interpolate(edge2, size=(h, w), mode='bilinear', align_corners=True)
        edge3 = F.interpolate(edge3, size=(h, w), mode='bilinear', align_corners=True)

        #***************
        edge = torch.cat([edge1, edge2, edge3], dim=1)  ### this edge not be used?
        #***************
        edge_fea = torch.cat([edge1_fea, edge2_fea, edge3_fea], dim=1)
        edge_fea = self.conv6(edge_fea)
        edge = self.conv5(edge_fea)

        return edge, edge_fea

**Comment by ziwei-zh** [comment]
Yes it is not used.
